src/client/workflow/serve.ts

```diff
@@ -66,6 +68,8 @@ const processOptions = <TResponse = Response, TInitialPayload = unknown>(
         currentSigningKey: process.env.QSTASH_CURRENT_SIGNING_KEY!,
         nextSigningKey: process.env.QSTASH_NEXT_SIGNING_KEY!,
       }),
+    failureUrl: false,
+    failureFunction: false,
```

Comment:
I would prefer to use undefined but the `processOptions` method makes everything (including failureFunction) required, so I can't make the type `failureFunction?: FailureFunction`. This is why I used `failureFunction?: FailureFunction | false`, to make it possible to pass a default fallsy value.

I wan't to be able to pass a fallsy value as `failureFunction` because I determine the failure callback URL by checking if `failureFunction` is set.

I can have a default string like "none" but `false` felt better to me

Reply:
Can't we just keep undefined as its default value? Tbh, that would make more sense.

Reply:
left them undefined using RequiredExceptFields